app/components/banner.tsx

```diff
@@ -0,0 +1,37 @@
+import { Trans } from "@lingui/react";
+import { Box } from "@mui/material";
+
+import { HintYellow } from "@/components/hint";
+
+import { useChartState } from "../charts/shared/chart-state";
+
+/**
+ * Banner to display errors or warnings for a chart. Each chart type can have specific warnings or errors depending on the data.
+ *
+ * @returns React.ReactNode | null
+ */
+export const Banner = () => {
+  const state = useChartState();
+
+  switch (state.chartType) {
+    case "area":
+      const someValueIsNegative = state.allData.some(
+        (d) => +(state.getY(d) || 0) < 0
+      );
```

Comment:
I think here we should use `state.chartData`, as the error should dynamically adjust to e.g. interactive filters (to not show it when some negative values are present in the whole dataset, but not in the current filter selection).